doc/doc_utils/jumpstart_doc_utils.py

```diff
@@ -74,9 +74,12 @@ class Frameworks(str, Enum):
 
 JUMPSTART_REGION = "eu-west-2"
 SDK_MANIFEST_FILE = "models_manifest.json"
+PROPRIETARY_SDK_MANIFEST_FILE = "proprietary-sdk-manifest.json"
 JUMPSTART_BUCKET_BASE_URL = "https://jumpstart-cache-prod-{}.s3.{}.amazonaws.com".format(
     JUMPSTART_REGION, JUMPSTART_REGION
 )
+PROPRIETARY_DOC_BUCKET = "https://jumpstart-cache-prod-us-west-2.s3.us-west-2.amazonaws.com"
```

Comment:
For open-weights models we point to a wave-4 region, `DUB` IIRC.

Could you use the same for consistency?

Reply:
I just found an issue with regionalization in some regions, raising a fix today. I'll update this to `DUB` in a separate PR to update documentation of `list_jumpstart_models`, and I'd like to refactor the doc utils in that PR as well